libcrux-nucleo-l4r5zi/memory.x

```diff
@@ -0,0 +1,12 @@
+MEMORY
+{
+    FLASH : ORIGIN = 0x08000000, LENGTH = 2048K /* BANK_1 */
+    RAM   : ORIGIN = 0x20000000, LENGTH =  640K /* SRAM */
+}
+
+
+/* This is where the call stack will be allocated. */
+/* The stack is of the full descending type. */
+/* You may want to use this variable to locate the call stack and static
+   variables in different memory regions. Below is shown the default value */
```

Comment:
Can you link the document that contains the notes here? https://md.cryspen.com/68n4HBCpR1qRe17_CRxiaQ?edit

Reply:
Good point! I've added a link there.